applications/GeoMechanicsApplication/tests/test_pressure_point_flux/Common/MaterialParameters2D.json

```diff
@@ -0,0 +1,30 @@
+{
+  "properties": [{
+    "model_part_name"                      : "PorousDomain.filter",
+    "properties_id"                        : 1,
+    "Material" : { 
+      "constitutive_law" : {
+        "name"                             : "GeoLinearElasticPlaneStrain2DLaw"
+      },
+      "Variables" : {
+        "IGNORE_UNDRAINED"                 : false,
+        "YOUNG_MODULUS"                    : 1.000000e+07,
+        "POISSON_RATIO"                    : 0.000000e+00,
+        "DENSITY_SOLID"                    : 2.650000e+03,
+        "DENSITY_WATER"                    : 1.000000e+03,
+        "POROSITY"                         : 1.000000e-01,
+        "BULK_MODULUS_SOLID"               : 9.000000e+19,
+        "BULK_MODULUS_FLUID"               : 1.000000e+20,
+        "PERMEABILITY_XX"                  : 2.000000e-04,
```

Comment:
The XX indicate a global coordinate system X direction, the line element is in global Y direction. Longitudinal permeability may be a better description here.

Reply:
@WPK4FEM  Thank you for review.
It will done on a new PR. I already opened a new issue for this for the thermal element, to move the checking of material properties to the constutative law. I Open another issue for this element, namely pw line element.

The reason of not removing PERMEABILITY_XX at the current stage, is to avoid "if-else" statements for 1D, 2D and 3D in the element level.